packages/config-utils/src/fec-logger.ts

```diff
@@ -0,0 +1,36 @@
+import chalk from 'chalk';
+
+const PREFIX = '[fec]';
+const ERROR_LEVEL = `${PREFIX} Error`;
+const WARN_LEVEL = `${PREFIX} Warn`;
+const INFO_LEVEL = `${PREFIX} Info`;
+const DEBUG_LEVEL = `${PREFIX} Debug`;
+
+export enum LogType {
+  error = ERROR_LEVEL,
+  warn = WARN_LEVEL,
+  info = INFO_LEVEL,
+  debug = DEBUG_LEVEL,
+}
+
+const colors = {
+  [LogType.error]: chalk.red,
+  [LogType.warn]: chalk.yellow,
+  [LogType.info]: chalk.blue,
+  [LogType.debug]: chalk.magenta,
+};
+
+const logFunctions = {
+  [LogType.error]: console.error,
+  [LogType.warn]: console.warn,
+  [LogType.info]: console.info,
+  [LogType.debug]: console.debug,
+};
+
+function getFecMessage(level: LogType, ...data: any[]) {
```

Comment:
@Hyperkid123 not sure what convention we use in this repo yet, but I typically try to avoid `any` where possible.

Reply:
It is just an extension of the `console.` typings. Generally using `any` is not good. That said if we only extend an existing interface we use the original types if possible.